colab/nessie-iceberg-spark-setup.ipynb

```diff
@@ -29,7 +29,7 @@
    "outputs": [],
    "source": [
     "# install the nessiedemo lib, which configures all required dependencies\n",
-    "!pip install nessiedemo\n"
+    "!pip install nessiedemo"
```

Comment:
I think we want this to be a description of how to set up Spark/Iceberg/Nessie. The `nessiedemo` library really should not be used. Or if it needs to be used there should be a *big* warning label that its only being called to facilitate the rest of the demo and to not use it IRL

Reply:
As the important part of this demo is to highlight the setup of the configs I think there should be much more discussion of the relevant parameters and their settings below (I can't comment below for some reason)